src/alchemlyb/workflows/abfe.py

```diff
@@ -115,7 +116,7 @@ def __init__(
         else:
             raise NotImplementedError(f"{software} parser not found.")
 
-    def read(self, read_u_nk=True, read_dHdl=True):
+    def read(self, read_u_nk=True, read_dHdl=True, n_jobs=-1):
```

Comment:
Is default -1 really always the best choice? Did you try on a machine with, say, 16 cores, and 16 hyperthreaded cores (or really anything with hyperthreads)?

I am willing to make -1 the default if this is not throwing surprises for users. Otherwise the conservative 1 would be better and users can then explicitly enable.

Reply:
add versionchanged to docs

Reply:
Add a paragraph about parallelization: how to enable it, what it does (for each file), any potential problems...